daemon/src/oracle.rs

```diff
@@ -38,6 +38,11 @@ pub struct MonitorAttestation {
     pub event_id: BitMexPriceEventId,
```

Comment:
Do you think it makes sense to get rid of `MonitorAttestation` in the favor of `MonitorAttestations` and just send around a vec with 1 element?

Reply:
I don't think it's a good idea for two reasons:

- The public API of this module doesn't need to let us send a message to monitor more than one attestation at a time.
- We would be allocating unnecessarily when sending `MonitorAttestations` with one item.

I think keeping both messages is worth it.